BookingsApi/BookingsApi.Domain/Hearing.cs

```diff
@@ -208,7 +212,28 @@ public Participant AddIndividual(Person person, HearingRole hearingRole, CaseRol
 
             return participant;
         }
+        
+        public Participant AddIndividual(string externalReferenceId, Person person, HearingRole hearingRole, string displayName)
```

Comment:
maybe a summary comment?

Reply:
I don't think it's necessary here. The method name is pretty clear

Reply:
What do you think about this?
If we add some information of what it is passed and used maybe it will be beneficial later on? If you think it is not necessary or it is too much no worries. 

        /// <summary>
        /// The AddIndividual function adds a new individual participant to a hearing, with optional
        /// audio recording requirement based on the participant's role and case type.
        /// </summary>
        /// <param name="externalReferenceId">The `externalReferenceId` parameter is a unique identifier
        /// for the individual participant, typically provided by an external system or source.</param>
        /// <param name="Person">A class representing a person with attributes such as name, date of
        /// birth, address, etc.</param>
        /// <param name="HearingRole">HearingRole is an enum that represents the role of a participant
        /// in a hearing. It could include values such as "Applicant", "Respondent", "Witness",
        /// "Interpreter", etc.</param>
        /// <param name="displayName">The `displayName` parameter in the `AddIndividual` method is a
        /// string that represents the name or title to be displayed for the individual participant in
        /// the context of a hearing. It is used to identify the participant within the system and may
        /// be shown in user interfaces or reports related to the hearing process.</param>
        /// <returns>
        /// The method `AddIndividual` is returning a `Participant` object.
        /// </returns>

Reply:
Honestly would reserve it for methods that are bit more complex. If we do it for this then we should look to be consistent and add it across the board